dreamjub/management/commands/ldapsync.py

```diff
@@ -1,22 +1,24 @@
-from django.core.management.base import BaseCommand, CommandError
-from jDREAM.models import Student
+from django.core.management import base
+from dreamjub.models import Student
 
 from getpass import getpass
 
-class Command(BaseCommand):
+
+class Command(base.BaseCommand):
     help = 'Reloads all students from LDAP. '
 
     def add_arguments(self, parser):
-        parser.add_argument('username', nargs='?', default=None, help='Username to use for LDAP. If omitted, will ask for username interactively. ')
-        parser.add_argument('password', nargs='?', default=None, help='Passowrd to use for LDAP. If omitted, will ask for password interactively. ')
+        parser.add_argument('username', nargs='?', default=None,
+                            help='Username to use for LDAP. If omitted, ' +
+                                 'will ask for username interactively. ')
+        parser.add_argument('password', nargs='?', default=None,
+                            help='Passowrd to use for LDAP. If omitted, ' +
+                                 'will ask for password interactively. ')
 
     def handle(self, *args, **options):
         if options["username"] is None:
             self.stdout.write("Username: ")
-            try:
-                user = input()
-            except NameError:
-                user = raw_input()
+            user = input()
```

Comment:
This was legacy code to fallback to Python2s raw_input() method. Since we are only supporting python3, it is no longer needed.